padinfo/padinfo.py

```diff
@@ -1512,30 +1512,34 @@ async def jpdgname(self, ctx, *, search_text):
 
     @commands.command(aliases=["jydl", "jpyt"], usage="<dungeon_name> / <monster_name>")
     async def jpyoutube(self, ctx, *, search_text):
-        """Attempt to link to a YouTube search of a leader in a dungeon"""
+        """Link to a YouTube search of a dungeon, with an option to specify leader"""
         dbcog = await self.get_dbcog()
         db: "DBCogDatabase" = dbcog.database.database
+
         if '/' in search_text:
```

Comment:
You could try something like:

If '/' in search_text or ',' in search_text:

and then try separating by one or the other